_data-prepper/common-use-cases/sampling.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Sampling
+parent: Common use cases
+nav_order: 45
+---
+
+# Sampling
+
+Data Prepper provides the following sampling capabilities:
+
+- Time sampling
+- Percentage sampling
+- Tail sampling
+
+## Time sampling 
+
+You can use the `rate_limiter` action within the [`aggregate` processor]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/aggregate/) to limit the number of events that can be processed per second. You can choose to either drop excess events or carry them forward to the next time period.
+
+In this example, only 100 events per second with a status code of `200` are sent to the sink from a given IP address. It drops all excess events from the configured time window.
+
+```json
+...
+  processor:
+   - aggregate:                                                                                                                                          
+        identification_keys: ["clientip"]                                                                                                      
+        action:                                                                                                                                           
+          rate_limiter:                                                                                                                                   
+            events_per_second: 100                                                                                                                        
+            when_exceeds: drop
+        when: "/status == 200"  
+...
+```
+
+If you instead set the `when_exceeds` option to `block`, the processor will block the pipeline until the time window completes. Then it will process the blocked events.
```

Comment:
```suggestion
If you instead set the `when_exceeds` option to `block`, the processor will block the pipeline until the time window has elapsed. Then it will process the blocked events.
```